templates/docs/section.html

```diff
@@ -28,8 +28,8 @@ <h1 class="text-center">{{ section.title }}</h1>
                 </div>
               </div>
             {% endfor %}
-            {# Have the newest version first #}
-            {% set subsections = index.subsections | sort | reverse %}
+            {# Sections sorted by weight #}
+            {% set subsections = index.subsections %}
```

Comment:
I don't follow, how we ensure that the order is correct without sorting ?

Reply:
IIRC the default ordering is by weight (a number specified in the frontmatter of each page/section), and our manual `sort` was overriding that and sorting by subsection name instead, which meant section ordering could only be changed by changing their names.